simpeg/electromagnetics/frequency_domain/simulation.py

```diff
@@ -64,24 +62,19 @@ class BaseFDEMSimulation(BaseEMSimulation):
     """
 
     fieldsPair = FieldsFDEM
-    permittivity = props.PhysicalProperty("Dielectric permittivity (F/m)")
 
     def __init__(
         self,
         mesh,
         survey=None,
+        *,
```

Comment:
I've done this to several classes to make it more clear in the source code which parameters should only be set as keywords.